### PR TITLE
General Homepage UI tweak.

### PR DESCRIPTION
**Pull request by MC42**
* Centered items on front pane of website: 
image
* Also reformatted slightly.

* Changed search to have an icon in it (also changed verbiage briefly to make it fit properly.)

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -72,7 +72,7 @@
         <input type="hidden" name="cx" value="017956276086362105630:99ctqytcokm" />
         <input type="hidden" name="cof" value="FORID:11" />
         <input type="hidden" name="ie" value="UTF-8" />
-        <input type="text" class="form-control typeahead" placeholder="Search for teams and events" name="q"/>
+        <input type="text" class="form-control typeahead" placeholder="🔎 Search teams and events" name="q"/>
```
Comment (fangeugene) [inline templates/base.html]: Does the icon actually show up? Try a glyphicon.

File: templates/index_lhc.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <input type="hidden" name="cx" value="017956276086362105630:99ctqytcokm">
   <input type="hidden" name="cof" value="FORID:11">
   <input type="hidden" name="ie" value="UTF-8">
-  <input type="text" class="form-control typeahead" placeholder="Search for teams and events" name="q">
+  <input type="text" class="form-control typeahead" placeholder="🔎 Search teams and events" name="q">
```
Comment (fangeugene) [inline templates/index_lhc.html]: Same as above.

File: templates_jinja2/base.html [hunk 3]
```diff
@@ -72,7 +72,7 @@
         <input type="hidden" name="cx" value="017956276086362105630:99ctqytcokm" />
         <input type="hidden" name="cof" value="FORID:11" />
         <input type="hidden" name="ie" value="UTF-8" />
-        <input type="text" class="form-control typeahead" placeholder="Search for teams and events" name="q"/>
+        <input type="text" class="form-control typeahead" placeholder="🔎 Search teams and events" name="q"/>
```
Comment (fangeugene) [inline templates_jinja2/base.html]: Same as above.

**Comment by MC42** [comment]
@fangeugene upon further review, the ASCII character doesn't always render, and to use a glyphicon in there is slightly messy.  I'll just back them out.

File: templates_jinja2/base.html (hunk 3 above)
Comment (MC42) [inline templates_jinja2/base.html]: Upon further review, the ASCII character doesn't always render on all platforms,  and to use a glyphicon in there is slightly messy. I'll just back them out.

File: templates/index_lhc.html (hunk 2 above)
Comment (MC42) [inline templates/index_lhc.html]: Same as above.

**Pull request merged**